WoWsShipBuilder.Web/Components/ShipSelector.razor

```diff
@@ -242,6 +245,12 @@
         {
             SelectedShips.Add(shipIndex);
         }
+        if (AllowCopies)
+        {
+#pragma warning disable BL0005
```

Comment:
Is this really necessary?

Reply:
It is, if you want to be able to select multiple of the same ships at once. Will be useful also for the ballistic charts later